### ISSUE TITLE
Apply transform to crop to keep in place

### ISSUE THREAD

**Issue by tdmorello** [BiAPoL/napari-crop#16]
Should a transform be applied to cropped output to keep them in place with the original? Maybe this would be an option?

**Comment by haesleinhuepf** [comment]
You mean something like `layer.translation = [crop_min_z, crop_min_y, crop_min_x]`, right?

**Comment by tdmorello** [comment]
yeah exactly -- so the cropped image stays in place in the viewer. In fact, maybe it should be the default behavior.
